setup.py

```diff
@@ -1,32 +1,19 @@
-import setuptools
+import pathlib
+from setuptools import setup
+import setuptools_scm
+
 
-install_requires = []
-tests_require = ["pytest", "pytest-cov", "pytest-flake8"]
-dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
 scm_version_template = """# Generated by setuptools_scm
 __all__ = ["__version__"]
 
 __version__ = "{version}"
 """
 
-setuptools.setup(
-    name="ts-fiber-spectrograph",
-    description="LSST Fiber Spectrograph",
+setup(
+    version=setuptools_scm.get_version(),
     use_scm_version={
         "write_to": "python/lsst/ts/FiberSpectrograph/version.py",
         "write_to_template": scm_version_template,
     },
-    setup_requires=["setuptools_scm", "pytest-runner"],
-    install_requires=install_requires,
-    package_dir={"": "python"},
-    packages=setuptools.find_namespace_packages(where="python"),
-    package_data={"": ["*.rst", "*.yaml"]},
-    scripts=["bin/run_FiberSpectrograph.py"],
-    tests_require=tests_require,
-    extras_require={"dev": dev_requires},
-    license="GPL",
-    project_urls={
-        "Bug Tracker": "https://jira.lsstcorp.org/secure/Dashboard.jspa",
-        "Source Code": "https://github.com/lsst-ts/ts_FiberSpectrograph",
-    },
+    scripts=[str(path) for path in pathlib.Path("bin").glob("*.py")],
```

Comment:
Is the plan to switch to using entry points instead of `bin/` scripts?

Reply:
I think the plan is to do that after cycle 26. In any case I suggest discussing this with Wouter and Tiago (I am happy to participate, but for now I'm just doing what I was told).

Reply:
It is a possibility that TSSW can consider doing now already instead of after cycle 26. Thanks for the suggestion, @timj!

Reply:
Done for cycle 26, thanks to a lot of work on Wouter's part, and some on mine as well.